src/portal/src/i18n/lang/en-us-lang.json

```diff
@@ -959,8 +957,11 @@
             "OIDC_SETNAMECONTENT": "You must create a Harbor username the first time when authenticating via a third party(OIDC).This will be used within Harbor to be associated with projects, roles, etc.",
             "OIDC_USERNAME": "Username",
             "GROUP_CLAIM_NAME": "Group Claim Name",
+            "GROUP_CLAIM_NAME_INFO": "The name of a custom group claim that you have configured in your OIDC provider",
             "OIDC_ADMIN_GROUP": "OIDC Admin Group",
-            "OIDC_ADMIN_GROUP_INFO": "Specify an OIDC admin group name. All OIDC users in this group will have harbor admin privilege. Keep it blank if you do not want to."
+            "OIDC_ADMIN_GROUP_INFO": "Specify an OIDC admin group name. All OIDC users in this group will have harbor admin privilege. Keep it blank if you do not want to.",
+            "OIDC_GROUP_FILTER": "OIDC Group Filter",
+            "OIDC_GROUP_FILTER_INFO": "The OIDC group filter which filters out the groups that don't match the regular expression."
```

Comment:
The description `OIDC_GROUP_FILTER_INFO ` is not clear, it can mean two things.
In the sentence there is a double negative: "filter out " and don't match.

Here are two alternatives.

1. Filter OIDC groups who match the provided regular expression. 
2. Exclude OIDC groups who match the provided regular expression.

Reply:
Updated as option 2